openbsd/OpenBSDProcessList.c

```diff
@@ -53,6 +57,13 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidMatchList, ui
       pl->cpuCount = 1;
    }
    opl->cpus = xCalloc(pl->cpuCount + 1, sizeof(CPUData));
+   opl->cpuIndex = xRealloc(opl->cpuIndex, pl->cpuCount * sizeof(int));
```

Comment:
`*opl` was initialized in this function, no need to realloc (just malloc (or calloc, see later comment))